allennlp/semparse/contexts/table_question_context.py

```diff
@@ -213,6 +210,88 @@ def read_from_lines(cls,
 
             table_data[-1][column_name] = cell_data
             last_row_index = row_index
+
+        return table_data, column_name_type_mapping
+
+    @classmethod
+    def get_table_data_from_untagged_lines(cls,
+                                           lines: List[List[str]]) -> Tuple[List[Dict[str, Dict[str, str]]],
+                                                                            Dict[str, Set[str]]]:
+        """
+        This method will be called only when we do not have tagged information from CoreNLP. That is, when we are
```

Comment:
Is it possible to just call corenlp's python API here instead of this?  Or would that require a call to java, which we're trying to avoid?  It probably would...

Reply:
I don't think we're missing out on a lot by not using CoreNLP here.